cmd/soroban-cli/src/commands/tx/mod.rs

```diff
@@ -0,0 +1,48 @@
+use clap::Parser;
+
+pub mod send;
+pub mod sign;
+pub mod simulate;
+pub mod xdr;
+
+use stellar_xdr::cli as xdr_cli;
+
+#[derive(Debug, Parser)]
+pub enum Cmd {
+    /// Add a new identity (keypair, ledger, macOS keychain)
+    Inspect(xdr_cli::Root),
+    /// Given an identity return its address (public key)
+    Sign(sign::Cmd),
```

Comment:
The doc comments on these two commands look like a copy-paste from somewhere else and don't make much sense in this context.